src/sider/3-soknadstilpasser/felles/personalia/Personalia.tsx

```diff
@@ -139,15 +146,19 @@ class VisPersonalia extends Component<MergedProps, { visError?: boolean }> {
                 )}
               </div>
               <Form className="personalia__paneler" autoComplete="off">
-                <FodselsnummerPanel />
-                <AdressePanel />
-                {personEllerBedrift === "bedrift" && (
+                <FodselsnummerPanel skalKlage={skalKlage} skalAnke={skalAnke} />
+                {!(skalKlage || skalAnke) && (
                   <>
-                    <FlerePersonerPanel />
-                    <TiltaksBedriftPanel />
+                    <AdressePanel />
```

Comment:
Skjul muligheter for ukjent-person, flere personer og tiltaksbedrifter for klage og anke

Reply:
^ Ser ukjent-person er skjult, men flere personer og tiltaksbedrifter ligger der? Og bør man i det hele tatt kunne velge klage eller anke på søknadene som ligger under bedrift? Kanskje det er enklere å toggle?

Reply:
Det ble første bestemt at klage skulle være aktivert for bedrift, men trur det ble en kontra på det. Så trur siste oppdatering er at vi lager en toggle som du foreslår :)